Testing/xASL_test_FullPipelineTest.m

```diff
@@ -0,0 +1,86 @@
+%% xASL_test_FullPipelineTest: BIDS testing script
```

Comment:
This looks familiar :), what is the difference between this script and the BIDS full pipeline test that was in `/Development/BIDS`? If this is the same one, please use `git mv` so it is "recorded" that this is not a new function but an adaption of an existing function

Reply:
I moved it, but the problem was the moving and renaming I guess. You don't really need me to revert all the changes because of that, right?

Reply:
Only for this file, yes. I would copy it, revert this comment, and use `git mv` and then `git add` for the changes. That should keep it nice. If it doesn't work, no big deal, but it is clearer for history (and for Jan and me to know what's happening :)